Assets/Scripts/Enemy.cs

```diff
@@ -0,0 +1,28 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[RequireComponent(
+    typeof(Rigidbody)
+)]
```

Comment:
This type makes no reference to any rigidbody. Suggest removing this `RequireComponent` attribute.